packages/app/src/static-props/get-data.ts

```diff
@@ -55,6 +67,37 @@ export function createGetContent<T>(
   };
 }
 
+/**
+ * This method returns all the national data that is required by the sidebar,
+ * optional extra metric property names can be added as separate arguments which will
+ * be added to the output
+ *
+ */
+export function selectNlPageMetricData<
+  T extends keyof National = NationalPageMetricNames
+>(...additionalMetrics: T[]) {
+  return selectNlData(...[...nationalPageMetricNames, ...additionalMetrics]);
```

Comment:
nit; Would this work too?

```suggestion
  return selectNlData(...nationalPageMetricNames, ...additionalMetrics);
```

Reply:
For some reason it doesn't, the compiler complains about it.